docs/sdk-and-tools/notifier.md

```diff
@@ -349,12 +363,30 @@ It is recommended to use the setup with RabbitMQ, if it is very important to avo
 
 [comment]: # (mx-context-auto)
 
+### WebSocket
+
+If WebSocket subscribing solution is selected via CLI parameter, an additional http
```

Comment:
_If the ... solution_, uppercase `HTTP`.

Reply:
done